packages/atomic/src/components/insight/atomic-insight-edit-toggle/atomic-insight-edit-toggle.tsx

```diff
@@ -16,11 +17,12 @@ export class AtomicInsightEditToggle {
 
   public render() {
     return (
-      <atomic-icon-button
+      <IconButton
+        style="outline-neutral"
         icon={EditIcon}
-        labelI18nKey="edit-insight"
-        click-callback={this.clickCallback}
-        tooltip={this.tooltip}
+        ariaLabel="Edit"
```

Comment:
should this be localized?

Reply:
Not necessary here. I

Insight panel in atomic is an "internal user only" component, not meant to be used directly by clients as of today, and so localization is not important at the moment.